config/i18n-tasks.yml

```diff
@@ -169,3 +169,4 @@ ignore_unused:
   - decidim.verifications.omniauth.authorizations.new.success
   - decidim.verifications.omniauth.errors
   - decidim.verifications.omniauth.errors.anti_affinity
+  - layouts.decidim.cookie_warning.*
```

Comment:
Shouldn't you remove the locales key rather than adding this line ?

Reply:
Yes I updated !